app/components/candidate_interface/apply_again_call_to_action_component.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="govuk-inset-text govuk-!-margin-top-0">
+  <h2 class="govuk-heading-m"><%= title %></h2>
+
+  <p class="govuk-body">If now’s the right time, you can still apply for courses that start this academic year.</p>
+
+  <p class="govuk-body">
+    To talk about your options, get in touch with a
+    <%= govuk_link_to('teacher training adviser', 'https://beta-adviser-getintoteaching.education.gov.uk/') %>">.
```

Comment:
![image](https://user-images.githubusercontent.com/42515961/113928507-d4006500-97e6-11eb-93ad-fbe204827152.png)


This is slightly off